ci-operator/step-registry/openshift-extended/test/longduration/openshift-extended-test-longduration-ref.yaml

```diff
@@ -11,7 +11,7 @@ ref:
         ETCD|PSAP|SDN|STORAGE|Authentication
         OLM|Network_Edge|Operator_SDK
         Image_Registry|Container_Engine_Tools
-        NODE|PerfScale|Cluster_Observability|Security_and_Compliance|LOGGING|CFE
+        NODE|PerfScale|Cluster_Observability|Security_and_Compliance|LOGGING|CFE|OTA
```

Comment:
Adding OTA here mean the other longduration jobs will also execute OTA test cases. Is it expected ?

Reply:
yes, it's expected. we also need ota cases executed in other jobs to cover more platforms

Reply:
if OTA is added as default, it is possible to cause general prow job to be timeout because OTA longduration case has 20 cases currently. maybe it should take long time. (more than 6hours?)

for such module, I suggest to add dedicated prow job for this module longduration, like other modules, Cluster_Infrastructure etc.

Reply:
if you still want to add it as default, I am ok. but need to monitor if the general longduration prow job meet timeout.
once it happen, need to move it out and create dedicated job for OTA.

I raise it because OTA longduration case should take long time which is possible to cause general longduration prow job timeout.

Reply:
Agree, we need to confirm if the new adding for OTA cases cause time issues.

Reply:
disruptive cases:
```
INFO[2024-08-07T09:16:08Z] Running step gcp-ipi-ovn-ipsec-f28-destructive-openshift-extended-test-disruptive. 
INFO[2024-08-07T11:10:28Z] Step gcp-ipi-ovn-ipsec-f28-destructive-openshift-extended-test-disruptive succeeded after 1h54m20s. 
```

non-disruptive cases:
```
INFO[2024-08-07T09:26:56Z] Running step azure-ipi-f28-openshift-extended-test. 
INFO[2024-08-07T10:07:38Z] Step azure-ipi-f28-openshift-extended-test succeeded after 40m41s. 
```

From above log, it looks like all longduration cases time will be less than 3 hours, so i think it's low risk to keep OTA in the default chain, right?

Reply:
@kuiwang02 wdyt

Reply:
Which tag OTA test cases use ?  I assume it has longduration tag because we are executing longduration chain in this PR. Shall we rehearse another longduration job to check the execution time ?

Reply:
Above disruptive cases+non-disruptive cases are the total non-disconnected ota cases, which include longduration cases. So the time will be less than above time.

```
$ grep -r "g\.It" extended/ota|grep -v DisconnectedOnly| wc -l
37

$ grep -r "g\.It" extended/ota|grep -v DisconnectedOnly |grep Longduration|wc -l
13
```

Reply:
@jiajliu 
1,
I just check the current general longduration job, some of them are almost 18 or 19h.
for example, https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-aws-ipi-f28-longduration-part1-3/1819792943418970112

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.15-arm64-nightly-aws-ipi-f28-longduration-part1-3/1817179977364279296

2, I checked rehearse  job https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55177/rehearse-55177-periodic-ci-openshift-verification-tests-master-ota-nightly-4.17-azure-ipi-f28/1821092844857724928 and find
there are two cases timeout
```console
[sig-updates] OTA cvo should Longduration-NonPreRelease-Author:dis-Medium-32138-cvo alert should not be fired when RetrievedUpdates failed due to nochannel [Serial][Slow]
[sig-updates] OTA cvo should Longduration-NonPreRelease-Author:jiajliu-Medium-41778-ClusterOperatorDown and ClusterOperatorDegradedon alerts when unset conditions [Slow]
```
so, the duration should not 40m if they are ok.
by the way, if you want them to pass, need to change 
```yaml
- as: azure-ipi-f28
  cron: 27 9 21 * *
  steps:
    cluster_profile: azure-qe
    env:
      BASE_DOMAIN: qe.azure.devcluster.openshift.com
      E2E_RUN_TAGS: '@ota'
      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
      TEST_SCENARIOS: OTA
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-azure-ipi
```
to
```yaml
- as: azure-ipi-f28
  cron: 27 9 21 * *
  steps:
    cluster_profile: azure-qe
    env:
      BASE_DOMAIN: qe.azure.devcluster.openshift.com
      E2E_RUN_TAGS: '@ota'
      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
      TEST_SCENARIOS: OTA
      TEST_TIMEOUT: "75"
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-azure-ipi
```

as summary, I agree OTA longduration duration is less than 3h even those two timeout cases pass, but it is possible to break some of existing general longduration job.
so,
1, I still suggest you to make dedicated job.
2, please correct job azure-ipi-f28 to avoid your case timeout

Reply:
I think longduration test cases are excluded (thus not executed) in normal/disruptive tests.

Reply:
> some of them are almost 18 or 19h

ok, if current time is almost 19h, i think we should not add more cases in the default chain. 

> I think longduration test cases are excluded (thus not executed) in normal/disruptive jobs/tests.

not really, longduration cases are executed in above normal.disrutive jobs(except for the timeout ones pointed by kui)
for example:
```
g.It("Longduration-NonPreRelease-Author:jianl-high-68397-CVO reconciles SCC resources which do not have release.openshift.io/create-only: true [Disruptive]" //passed in disruptive [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55177/rehearse-55177-periodic-ci-openshift-verification-tests-master-ota-nightly-4.17-gcp-ipi-ovn-ipsec-f28-destructive/1821092269315330048) 

g.It("Longduration-NonPreRelease-Author:evakhoni-Medium-22641-Rollback against a dummy start update with oc adm upgrade clear [Serial] //passed in normal [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55177/rehearse-55177-periodic-ci-openshift-verification-tests-master-ota-nightly-4.17-azure-ipi-f28/1821092844857724928)
```

Reply:
From above, let me drop OTA from the list, considering we might add more ota dedicated jobs in the future to cover more platforms, so i plan to move the dedicated jobs from current `verification-tests` repo to `openshift-tests-private` repo to reduce the maintenance work on this new yamls which we thought only copy/paste work in the beginning. cc @jianlinliu

Reply:
Sounds good to me, thanks.

Reply:
Move to `openshift-tests-private` repo and drop OTA from default longduration chain.